sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/Capabilities.java

```diff
@@ -37,6 +56,10 @@ public boolean supportsCNINetworking() throws IOException {
         return hasOrExceedsVersion(1, 9);
     }
 
+    public boolean supportsPreReservedResources() {
+        return false;
```

Comment:
Should this be checking 1.10? If it's still not in DCOS by default, could put it behind SchedulerFlags/an envvar to allow testing with the same build.